pkg/config/options.go

```diff
@@ -4,12 +4,20 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"os/exec"
+	"strings"
 
 	"github.com/sirupsen/logrus"
 
+	"k8s.io/apimachinery/pkg/util/sets"
+
 	cioperatorapi "github.com/openshift/ci-tools/pkg/api"
 )
 
+const (
+	mainBranch = "master"
```

Comment:
Let's get `clonerefs` to set configure the upstream tracking branch and use:

```
git rev-parse --abbrev-ref --symbolic-full-name @{upstream}
```

To figure this out dynamically.

Reply:
This doesn't work in my environment though.

Reply:
Btw, most of the tools won't work in a different repo than `release`. So the value will always be `master` until we change it.

Reply:
What does that do in your environment? Have you configured an upstream tracking branch?

I don't want us to hard-code this since we're going to get bitten by the cost of dealing with it once `master` changes to `main`

Reply:
Updated, but I still have one concern. When a user will `make jobs` we want to operate on a subset of the changes, but this won't be possible since the user has to configure an @{upstream} in his local git release repo, right?